source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -453,13 +453,16 @@ Establishing a Connection (Internal Implementation)
 
 Before a `Connection <#connection>`_ can be marked as either "available" or "in use", it
 must be established. This process involves performing the initial
-handshake, handling OP_COMPRESSED, and performing authentication.
+handshake, inspecting the value of ``maxWireVersion`` (to decide whether to
+use ``OP_MSG``(>=3.6) or ``OP_QUERY``(<3.6) for future communications),
```

Comment:
Here and in the change in OP_MSG you should say what the relevant wire version is.

Reply:
Done.